contributing/calendar-project-management.qmd

```diff
@@ -0,0 +1,46 @@
+## GitHub for Project Management and Calendaring
+
+People want to be able to see ongoing NASA Openscapes projects, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.
+
```

Comment:
I'd suggest putting the screenshot up here: (show not tell, early on) - with the new screenshot from today's This Week / This Quarter / This Year